src/arraymancer/tensor/operators_blas_l1.nim

```diff
@@ -44,8 +43,33 @@ proc dot*[T: SomeInteger](a, b: Tensor[T]): T {.noSideEffect.} =
 
 proc `+`*[T: SomeNumber|Complex[float32]|Complex[float64]](a, b: Tensor[T]): Tensor[T] {.noinit.} =
   ## Tensor addition
+  #echo "+++++ SAME ++++++"
   map2_inline(a, b, x + y)
 
+proc `+`*[T, K: SomeNumber](a: Tensor[Complex[T]], b: Tensor[K]): Tensor[Complex[T]] {.noinit.} =
+  ## Tensor addition
+  echo "+++++ COMPLEX ++++++"
+  when T is K:
+    echo "T is K"
+    map2_inline(a, b, x + y)
+  elif T is float64:
+    echo "T is float64"
+    map2_inline(a, b, x + float64(y))
+  else:
+    {.error: "Tensor addition not supported for tensors of types " & $T & " and " & $K}
+
+proc `+`*[T, K: SomeNumber](a: Tensor[K], b: Tensor[Complex[T]]): Tensor[Complex[T]] {.noinit.} =
+  ## Tensor addition
+  echo "+++++++++++2"
+  when T is K:
+    echo "T is K"
+    map2_inline(a, b, x + y)
+  elif T is float64:
+    echo "T is float64"
+    map2_inline(a, b, float64(x) + y)
+  else:
+    {.error: "Complex tensor addition only supported for tensors of the same type."}
+
```

Comment:
I guess this is part of this PR by accident?

Reply:
Definitely, sorry about that. I'll remove this spurious change and update the PR.